week08_pomdp/practice_pytorch.ipynb

```diff
@@ -582,7 +649,10 @@
     "moving_average = lambda x, **kw: DataFrame(\n",
     "    {'x': np.asarray(x)}).x.ewm(**kw).mean().values\n",
     "\n",
-    "rewards_history = []"
+    "set_seed()\n",
+    "rewards_history, grad_norm_history, entropy_history, entropy_loss_history, loss_history, Jhat, v_loss = [],[],[],[],[],[],[]\n",
+    "agent = SimpleRecurrentAgent(obs_shape, n_actions)\n",
+    "opt = torch.optim.Adam(agent.parameters(), lr=1e-5)\n"
```

Comment:
Why nonstandard `lr`?

Reply:
What??? It is?!!!

![image](https://user-images.githubusercontent.com/72295771/205723420-378d060c-e444-4b53-98eb-dece2fc1c9e4.png)